bolt/lib/Passes/PAuthGadgetScanner.cpp

```diff
@@ -490,6 +514,11 @@ class SrcSafetyAnalysis {
       dbgs() << ")\n";
     });
 
+    // Being trusted is a strictly stronger property than being
+    // safe-to-dereference.
+    assert(!Next.TrustedRegs.test(Next.SafeToDerefRegs) &&
+           "SafeToDerefRegs should contain all TrustedRegs");
+
```

Comment:
`BitVector::test(const BitVector &)` name looks rather ambiguous to me, but here is its definition: [BitVector.h](https://github.com/llvm/llvm-project/blob/5bb8e9db5f59dc2d3d8658f5b03577f9296be267/llvm/include/llvm/ADT/BitVector.h#L536).

Reply:
Proposed `BitVector::subsetOf` in #170875.